cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,17 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  const email = 'qayf@test.com';
```

Comment:
Hard-coded email and password are not a good practice. It should be moved to a config or environment file.